tests/Unit/Elliptic/Actions/Test_InitializeSystem.cpp

```diff
@@ -0,0 +1,249 @@
+};
+
+template <size_t Dim>
+void check_compute_items(
+    const ActionTesting::MockRuntimeSystem<Metavariables<Dim>>& runner,
+    const ElementId<Dim>& element_id) {
+  // The compute items themselves are tested elsewhere, so just check if they
+  // were indeed added by the initializer
+  const auto tag_is_retrievable = [&runner,
+                                   &element_id](auto tag_v) -> decltype(auto) {
+    using tag = std::decay_t<decltype(tag_v)>;
+    return ActionTesting::tag_is_retrievable<
+        ElementArray<Dim, Metavariables<Dim>>, tag>(runner, element_id);
+  };
+  CHECK(
+      tag_is_retrievable(::Tags::deriv<SomePrefix<ScalarFieldTag>,
+                                       tmpl::size_t<Dim>, Frame::Inertial>{}));
+}
+
+}  // namespace
+
+SPECTRE_TEST_CASE("Unit.Elliptic.Actions.InitializeSystem",
+                  "[Unit][Elliptic][Actions]") {
+  {
+    INFO("1D");
+    // Which element we work with does not matter for this test
+    const ElementId<1> element_id{0, {{SegmentId{2, 1}}}};
+    const domain::creators::Interval<Frame::Inertial> domain_creator{
+        {{-0.5}}, {{1.5}}, {{false}}, {{2}}, {{4}}};
+    // Register the coordinate map for serialization
+    PUPable_reg(
+        SINGLE_ARG(domain::CoordinateMap<Frame::Logical, Frame::Inertial,
+                                         domain::CoordinateMaps::Affine>));
+
+    using metavariables = Metavariables<1>;
+    using element_array = ElementArray<1, metavariables>;
+    ActionTesting::MockRuntimeSystem<metavariables> runner{
+        {AnalyticSolution<1>{}}};
+    ActionTesting::emplace_component_and_initialize<element_array>(
+        &runner, element_id,
+        {domain_creator.create_domain(), domain_creator.initial_extents()});
+    ActionTesting::next_action<element_array>(make_not_null(&runner),
+                                              element_id);
+    runner.set_phase(metavariables::Phase::Testing);
+    ActionTesting::next_action<element_array>(make_not_null(&runner),
+                                              element_id);
+    const auto get_tag = [&runner, &element_id](auto tag_v) -> decltype(auto) {
+      using tag = std::decay_t<decltype(tag_v)>;
+      return ActionTesting::get_databox_tag<element_array, tag>(runner,
+                                                                element_id);
+    };
+
+    check_compute_items(runner, element_id);
+
+    CHECK(get_tag(ScalarFieldTag{}) == Scalar<DataVector>{{{{4, 0.}}}});
+    CHECK(get_tag(LinearSolver::Tags::OperatorAppliedTo<ScalarFieldTag>{}) ==
+          Scalar<DataVector>{{{{4, 0.}}}});
+    CHECK(get(get_tag(Tags::Source<ScalarFieldTag>{})) ==
+          get<0>(get_tag(Tags::Coordinates<1, Frame::Inertial>{})));
+    CHECK(get(get_tag(SomePrefix<ScalarFieldTag>{})).size() == 4);
+    CHECK(get(get_tag(LinearSolver::Tags::OperatorAppliedTo<
+                      SomePrefix<ScalarFieldTag>>{}))
+              .size() == 4);
```

Comment:
I replaced that test with a check that the deriv tag `is_retrievable` (in `check_compute_items` above). That’s because the compute item has its own test that makes sure it works. I’m trying to move away from repeating tests.